packages/core/upload/admin/src/components/AssetList/Draggable.js

```diff
@@ -0,0 +1,49 @@
+import React, { useRef } from 'react';
+import PropTypes from 'prop-types';
+import { useDrag, useDrop } from 'react-dnd';
+
+export const Draggable = ({ children, id, index, moveItem }) => {
+  const ref = useRef(null);
+
+  const [, drop] = useDrop({
+    accept: 'draggable',
+    hover(hoveredOverItem) {
+      if (!ref.current) {
+        return;
+      }
+
+      if (hoveredOverItem.id !== id) {
+        moveItem(hoveredOverItem.index, index);
+
+        hoveredOverItem.index = index;
+      }
+    },
+  });
+
+  const [{ isDragging }, drag] = useDrag({
+    type: 'draggable',
+    item: () => {
+      return { index, id };
+    },
+    collect: monitor => ({
+      isDragging: monitor.isDragging(),
+    }),
+  });
+
+  const opacity = isDragging ? 0 : 1;
```

Comment:
Maybe it's a bit too strong setting the opacity to 0, what do you think about `0.2 : 1` ?